tests/kafkatest/services/templates/mirror_maker_consumer.properties

```diff
@@ -16,9 +16,11 @@
 
 {% if new_consumer %}
 bootstrap.servers={{ source.bootstrap_servers(security_config.security_protocol) }}
+auto.offset.reset={{ source_auto_offset_reset|default('latest') }}
```

Comment:
Also seems unrelated? And this setting doesn't appear in the service anywhere? I think it's fine to have sections of the template that use extra fields if they are assigned, but then they probably need docs somewhere. It'd probably be good to standardize on either passing all parameters through the constructor (even if it uses `**kwargs` to avoid enumerating them all in the constructor's signature) or assigning fields on the Service object after it is created.